src/index.html

```diff
@@ -6,12 +6,185 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <section class="section">
+          <label for="surname">Surname:</label>
+          <input
+            required
+            id="surname"
+            autocomplete="off"
+            type="text"
+            name="surname"
+          >
+        </section>
```

Comment:
```suggestion
      <fieldset class="fields">
        <legend>Personal information</legend>
        
        <section class="section">
          <label for="surname">Surname:</label>
          
          <input
            required
            id="surname"
            autocomplete="off"
            type="text"
            name="surname"
          >
        </section>
```
Add new lines between adjacent elements. Please fix all cases